tech/languages/python/scipy.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Scientific Python Stack
+subsection: python
+order: 9
+---
+
+# Scientific Python Stack
+
+There is a lot of libraries for scientific computation and visualization
+available in Fedora. Well known and widely used is
+[SciPy Stack](v) which consists of
+
+* [Python](https://docs.python.org/3/), a general purpose object-oriented
+  programming language,
+* [NumPy](https://docs.scipy.org/doc/numpy/), a Python library providing
+  fast multidimensional arrays with vector operations,
+* [SciPy](https://docs.scipy.org/doc/scipy/reference/), a Python library
+  providing computational routines for e.g. numerical integration, various
+  equation solvers and optimization.
+* [matplotlib](http://matplotlib.org/), a powerful Python library providing
+  scientific visualization of publication quality,
+* [IPython](https://ipython.org/), an enhanced interactive Python interpreter,
+* [pandas](http://pandas.pydata.org/), a Python library providing data type
+  for data series manipulation,
+* [SymPy](http://www.sympy.org/en/index.html), a Python library for computer algebra
+  support (i.e. symbolical computation) and
+* [nose](http://nose.readthedocs.io/en/latest/), a testing framework for Python.
+
+## Numpy
+
+[NumPy](http://www.numpy.org/) is a high performance Python library providing
+fast multidimensional arrays featuring vector operations. What are vector
+operations? Those which are applied on multiple values at once, see:
+
```

Comment:
I would prefer to move there an installation via `dnf` and afterwards explain how to work with `Numpy`